packages/cli/src/commands/build/index.ts

```diff
@@ -669,7 +669,7 @@ async function doBuild(
     overrides.length > 0 ? Object.assign({}, ...overrides) : undefined;
   const mergedFlags = mergeFlags(buildResults.values());
 
-  const framework = await getFramework(cwd, buildResults);
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** add test

Is there a test we can add for this?‏

Reply:
Test added in 5681aa72ce5cf87958e4741a0c30fdc971e8fa9e.